gob/k8s/README.md

```diff
@@ -235,7 +236,7 @@ We can test out our staged service (without altering the web service
 at all), by adding a delegation (routing rule) to a request.  For example:
 
 ```
-:; curl -H 'Dtab-local: /host/gen => /srv/gen-growthhack' "$GOB_HOST/gob?text=gob&limit=10"
+:; curl -H 'Dtab-local: /srv/gen => /srv/gen-growthhack' "$GOB_HOST/gob?text=gob&limit=10"
 bees <3 k8s
```

Comment:
Suuuper minor, but the output here doesn't reflect the curl output correctly. Since the curl command is made with `text=gob`, then we should see `gob <3 k8s`, instead of `bees <3 k8s`.